site/content/contribute/more-info/server/developer-setup/windows-wsl.md

```diff
@@ -125,3 +126,6 @@ This is an unofficial guide. Community testing, feedback, and improvements are w
     If that's the case, `postgresql.conf` is probably a directory rather than a file, which means the volume wasn't mounted properly, so you are most likely missing some of the configuration changes from step 1 (Linux Subsystem Installation).
     
 4. If you see an error like `ERROR: for mattermost-postgres. Cannot create container for service postgres: status code not OK but 500: {"Message":"Unhandled exception: Drive has not been shared"}`, make sure you enabled file sharing for the drive that contains the `mattermost-server` project.
+
+5. If you see an error like `go: could not create module cache: mkdir /home/<Linux User>/go: file exists
+Top-level object must be a mapping`, rename the workspace from step 5 part 2 to not be "go".
```

Comment:
@WilliamLongKing @cwarnermm Could we give an example path, This is `not to be 'go'`. Is not clear enough.

Reply:
Thanks, @AshishDhama. I agree that what's there now isn't clear. @WilliamLongKing - Are you open to providing an example path in this PR?

Reply:
@cwarnermm @AshishDhama I believe the path I'm referring to is the one from the previous sentence. I can make that more explicit?

Reply:
@WilliamLongKing - If I'm understanding the context correctly, we're providing troubleshooting suggestions for specific error conditions, and that in this case, if you see an error that looks like "this", you need to take specific action.

What's challenging for the reader is having to understand precisely what's being referred to (step 5 part 2) in combination with the action that the user needs to take.

As I expanded the source code in this PR beyond just the diff to include the rest of the file context, I'm not 100% clear where to look when told to look at "step 5 part 2" -- partly because I think there's some numbering issues with the steps in the code currently, and partly because the "part 2" aspect isn't obvious (to me).

Can you point me to the content on the page that aligns to "step 5 part 2" please? From there, I'll be able to provide some additional guidance.

Reply:
@WilliamLongKing - I have a few recommendations for moving forward:

1. In step #5, change the sub-steps numbered 1 & 2 to bullets. We're offering both Powershell and bash details. The user doesn't need to and shouldn't complete both steps. They only need to complete one or the other. While probably assumed and implied, it's well worth being explicit to guide users towards a successful outcome.
![Screenshot 2022-12-14 at 4 33 00 PM](https://user-images.githubusercontent.com/74422101/207720010-055fc1b8-1a38-4000-a586-e50ac18ce61e.png)

2. Under Troubleshooting, please remove the numbered steps altogether. This time you won't be replacing the numbers with bullets. Each troubleshooting item can stand alone as an individual question/answer pair.

3. For the final troubleshooting item, avoid forcing the user to leave the troubleshooting section to find the information they need to solve their problem. Instead, expand on the details the user needs to know, while remaining in the Troubleshooting section, to be successful.

Something along the lines of: 

If you see an error like `go: could not create module cache: mkdir /home/<Linux User>/go: file exists Top-level object must be a mapping`, rename the workspace to something other than "go" using the following bash command:

ln -s "/mnt/d/Projects/newworkspace" /home/<Linux User>/newworkspace